src/Components/Admin/AssetModelViewer/assetModelViewer.js

```diff
@@ -122,18 +124,27 @@ export default function AssetModelViewer({
                         onChange={handleNameChange}
                         variant="outlined"
                     />
-                    
-                    <FormControl variant="outlined" className={classes.formControl}>
-                        <InputLabel id="objectTypeLabel">Object Type</InputLabel>
+
+                    <FormControl
+                        variant="outlined"
+                        className={classes.formControl}
+                    >
+                        <InputLabel id="objectTypeLabel">
+                            Object Type
+                        </InputLabel>
                         <Select
                             labelId="objectTypeTextField"
                             label="Object Type"
                             id="demo-simple-select-helper"
                             value={objectType}
                             onChange={handleObjectTypeChange}
                         >
-                            <MenuItem value="object">{ObjectTypes.OBJECT}</MenuItem>
-                            <MenuItem value="background">{ObjectTypes.BACKGROUND}</MenuItem>
+                            <MenuItem value="object">
+                                {ObjectTypes.OBJECT}
+                            </MenuItem>
+                            <MenuItem value="background">
+                                {ObjectTypes.BACKGROUND}
+                            </MenuItem>
```

Comment:
I don't know if this is worth the change, but we could do `<MenuItem value={ObjectTypes.BACKGROUND}>...`. Definitely a nit and this shouldn't block you from merging (feel free to dismiss this feedback)